.github/workflows/deploy-preview.yml

```diff
@@ -66,6 +66,9 @@ jobs:
 
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         if: github.event.action == 'opened' || github.event.action == 'synchronize'
+        with:
+          sparse-checkout-cone-mode: false # exclude root files
+          sparse-checkout: docs
```

Comment:
We can make this configurable in the future if we like but every standard project and every one that I can think of (ignoring plugins) uses this structure.

Reply:
Great, that means I can get rid of trimming the `source_directory`.